docs/api.rst

```diff
@@ -25,6 +25,17 @@ Types
 
 .. todo:: Add links to the upstream documentation for the GA4GH types.
 
+
```

Comment:
Please remove this section --- this will just print out confusing protobuf stuff into the docs.